### ISSUE TITLE
Display_error warning stay red in installation php.ini check even if right in php.ini

### ISSUE THREAD

**Issue by baelmyhu** [chamilo/chamilo-lms#2230]
### Current behavior / Resultado actual / Résultat actuel
In php.ini display_errors = Off
and installation page show Display Error = On
see (https://content.screencast.com/users/hborderi/folders/Jing/media/cd7b7a98-1c69-4e58-9acb-584919954f37/2017-11-28_1642.png)
Same for entry Memory limit
see https://content.screencast.com/users/hborderi/folders/Jing/media/65a343ad-1ea1-4fbf-b4c2-1e4da3b0ed33/2017-11-28_1650.png

### Expected behavior / Resultado esperado / Résultat attendu
Display Error = Off in green for current installation

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.6

**Comment by baelmyhu** [comment]
[Debian GNU/Linux 7.11 (wheezy)]
Actually, for display_errors, 
if it is Off, ini_get('display_errors') = "", 
and if it is On  ini_get('display_errors') = "1"

For memory_limit, you're right, the install script put it to "-1"

**Comment by jmontoyaa** [comment]
I cannot reproduce the error. 
"display_errors" can be setup in many different places ... 
Check your virtual host.

**Comment by mrik974** [comment]
I was having the same problem and I think I found why.
The installation page calls `/main/install/index.php` which starts with a line saying `ini_set('display_errors', '1');`

**Comment by NicoDucou** [comment]
I can reproduce the problem and mrik974 is right it comes from the ini_set at the beginning of main/install/index.php that makes the check always red on the prerequisit test. 
So I think we should or remove it or change the system check because it always gets red and let people think there is something wrong and stop the installation process.

```
diff --git a/main/install/index.php b/main/install/index.php
index f8e094e165..75876f6189 100755
--- a/main/install/index.php
+++ b/main/install/index.php
@@ -17,9 +17,9 @@ use ChamiloSession as Session;
  * @package chamilo.install
  */
 
-ini_set('display_errors', '1');
+//ini_set('display_errors', '1');
 ini_set('log_errors', '1');
-ini_set('memory_limit', -1);
+//ini_set('memory_limit', -1);
 ini_set('max_execution_time', 0);
 error_reporting(-1);

```
When I comment those 2 lines I get the correct value of my system and all is green for the install.
@jmontoyaa  why do we need those ini_set ?

**Comment by july8six** [comment]
I am also experiencing this issue.  PHP 7.1

**Comment by ywarnier** [comment]
Did anyone test an installation recently to confirm that the issue is gone?

**Comment by NicoDucou** [comment]
I just did a new installation and everything is fine.
I close this task

**Comment by jorgemop** [comment]
same error on 1.11.6 

![image](https://user-images.githubusercontent.com/12866116/43419824-758af398-9418-11e8-818d-60f04da1c269.png)

**Comment by ywarnier** [comment]
@jorgemop this is fixed in the development version, to be published as v1.11.8. This is not yet available. We cannot change the 1.11.6 package after having published it. You can either download the development version (see README for 1.11.x branch) or wait for 1.11.8 to be released.
